ecosystem/sep-0006.md

```diff
@@ -589,29 +793,39 @@ The response should be a JSON object like:
 }
 ```
 
-The JSON object contains an entry for each Stellar asset that the anchor supports for SEP6 deposit and/or withdrawal.
+The JSON object contains an entry for each Stellar asset that the anchor supports for SEP-6 deposit and/or withdrawal.
+
+#### Response attributes for deposit assets
 
-#### For each deposit asset, response contains:
+All assets listed in a `deposit` and `deposit-exchange` can contain these attributes:
 
 * `enabled`: `true` if SEP-6 deposit for this asset is supported
 * `authentication_required`: Optional. `true` if client must be [authenticated](#authentication) before accessing the deposit endpoint for this asset. `false` if not specified.
 * `min_amount`: Optional minimum amount. No limit if not specified.
 * `max_amount`: Optional maximum amount. No limit if not specified.
```

Comment:
These attributes shouldn't be valid in `deposit-exchange` objects because the asset under which they're provided is the on-chain asset, and the user provides the off-chain asset.

Reply:
Should we update the SEP-38 description to enforce min_amount and max_amount when providing quotes?

Reply:
The anchor is free to return 400 Bad Request for `POST /quote` or `GET /price` requests with too small of an amount as the SEP is now, but it may be helpful to add `min_amount`/`max_amount` to either `GET /info` or `GET /prices` for the client's convenience. I don't feel strongly either way.